website/docs/guides/dynamic-assets.mdx

```diff
@@ -1,5 +1,13 @@
 # Dynamic Assets
 
+:::info
+
+This does not work with vite v5.0.0+ and wails v2. 
+Changes are planned in v3 to support this again.
```

Comment:
We don't plan to support the AssetsHandler anymore on v3. There will only be the `AssetsMiddleware` on v3.

Maybe it would be also good to mention it [here](https://wails.io/docs/reference/options#handler) because there's also note for limited support with vite.

Reply:
thanks @stffabi ,
two new commits to reword the note and copy it to the area suggested